integration_tests/serverless.yml

```diff
@@ -107,6 +126,16 @@ functions:
       DD_LOG_LEVEL: DEBUG
       DD_LAMBDA_HANDLER: src/metricPython.timeout
 
+  timeout-go:
+    runtime: provided.al2
+    timeout: 3
+    handler: src/bin/timeout
+    layers:
+      - { Ref: RecorderExtensionLambdaLayer }
+      - { Ref: DatadogExtensionIntegrationTestLambdaLayer }
+    environment:
+      DD_LOG_LEVEL: DEBUG
```

Comment:
Why do you want to set the log level to `DEBUG` for this test?

Reply:
I just followed the other tests. Not sure that it matters.